packages/fluentui/react-northstar-prototypes/src/prototypes/MenuList/Menu.tsx

```diff
@@ -3,7 +3,7 @@ import { MenuContextProvider } from './menuContext';
 import { useEventListener } from '@fluentui/react-component-event-listener';
 
 export const Menu = props => {
-  const { children, trigger = null, open = false } = props;
+  const { children, trigger = null, open = false, onCheckedValuesChange } = props;
```

Comment:
Can you please use `useAutoControlled()` hook to implement `defaultCheckedValues` behavior